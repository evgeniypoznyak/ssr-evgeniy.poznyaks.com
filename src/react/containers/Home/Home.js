import React from 'react';
import {makeStyles} from '@material-ui/core/styles';
import Avatar from '@material-ui/core/Avatar';
import Grid from '@material-ui/core/Grid';
import {Bio} from '../../shared/initialData/bio';

const useStyles = makeStyles(theme => ({

    avatar: {
        margin: 10,
    },
    bigAvatar: {
        margin: 20,
        width: 100,
        height: 100,
    },
}));

const Home = props => {
    const classes = useStyles();
    return (
        <div>
            <Grid container justify="flex-end">
                <Avatar alt="Evgeniy Poznyak" src={'/assets/me.jpg'} className={classes.bigAvatar}/>
            </Grid>
            {Bio}
        </div>
    );
};

export default Home;
