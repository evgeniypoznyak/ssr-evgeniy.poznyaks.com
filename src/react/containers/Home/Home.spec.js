import React from 'react';
import Home from './Home';
import renderer from 'react-test-renderer';
import {mount, shallow} from 'enzyme';

const mountWrapper = mount(<Home/>);
const wrapper = shallow(<Home/>);

describe('<Home/>', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<Home/>).toJSON();
        expect(tree).toMatchSnapshot();
        expect(tree.children.length).toBe(2);
    });

    it('should render correctly with enzyme', () => {
        expect(wrapper.getElement()).toMatchSnapshot();
        expect(wrapper.getElement().props.children.length).toBe(2);
    });

    it('should render correctly with mount wrapper', () => {
        expect(mountWrapper.getElement()).toMatchSnapshot();
        expect(mountWrapper.children.length).toBe(1);
    });
});
