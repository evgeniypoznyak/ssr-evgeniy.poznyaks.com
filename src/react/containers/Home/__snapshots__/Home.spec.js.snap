// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home/> renders correctly 1`] = `
<div>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-flex-end"
  >
    <div
      className="MuiAvatar-root makeStyles-bigAvatar-2"
    >
      <img
        alt="Evgeniy Poznyak"
        className="MuiAvatar-img"
        src="/assets/me.jpg"
      />
    </div>
  </div>
  Let me tell you a little about myself and my company to better introduce myself so you can all get to know me a little better.

About the company name. E&K photography was easily named after my first initial as well as my wife’s first initial. She is my biggest helper, assistant, secretary - you name it! All scheduling and questions usually go through her anyways. I am just the photographer that gets the job done and captures life’s beautiful moments my clients gift me.

A bit about myself. Where does photography come into play in my life? Where did I even begin with it? Photography was part of my life since early childhood. My father had a film camera that highly interested me and by the age of 10 my father bought me my first SLR camera. Kids my age were into playing with toys, but I was more interested in a real camera and taking pictures. My first pictures were on black and white film and were developed in a dark room. I started out taking pictures of everything around me. Nature, life events… anywhere I’d go I’d have a camera on hand and my friends were always my photo models. As time would pass friends would ask me to photograph their weddings, with my love for photography, I was excited to be given such opportunities.

I also had different full time and side jobs as a photographer when my wife and I lived in New York City. Within the years, more professional and personal opportunities became available to me opening more interest and experience for me to excel with my love for photography. I came to the point where I really wanted to take my photography more seriously as opposed to a hobby. I began to watch educational videos and read a lot of books on different tips and tricks on lights, and flashes and editing. Everything that would better me as a photographer and help me branch out. This is where I am now.

I now have acquired a professional camera - Nikon D810 - what I consider the best of the best and it has allowed me to excel so much more than I have ever done so before. More opportunities have come for me. From newborn sessions, weddings, engagement sessions, first birthday sessions, maternity photos, family photos, birthdays and variety of events… you name it. I enjoy taking such pictures. Being able to capture a single moment that sometimes you can never relive again. Freezing time in one shot. I love that I am able to do that, and help people treasure such moments and memories. My opinion is that as a photographer, one must always have room for improvement. One must always strive to better and better his work and art. New ideas have to constantly keep coming to life. Even if you become an amazing photographer at the peak of your career, you must always leave room for improvement and try to better yourself. I love my work, and I love what I do. I think that is the key to any career or anything you do in life. Every client I meet is such an honor for me to be able to get a glimpse into their life and be able to capture beautiful moments. So with that being said, I will end this saying that I put a piece of my soul into every photo I take, and I hope that if you do choose me as your photographer - I hope to never disappoint.
</div>
`;

exports[`<Home/> should render correctly with enzyme 1`] = `
<div>
  <ForwardRef(WithStyles)
    container={true}
    justify="flex-end"
  >
    <ForwardRef(WithStyles)
      alt="Evgeniy Poznyak"
      className="makeStyles-bigAvatar-2"
      src="/assets/me.jpg"
    />
  </ForwardRef(WithStyles)>
  Let me tell you a little about myself and my company to better introduce myself so you can all get to know me a little better.

About the company name. E&K photography was easily named after my first initial as well as my wife’s first initial. She is my biggest helper, assistant, secretary - you name it! All scheduling and questions usually go through her anyways. I am just the photographer that gets the job done and captures life’s beautiful moments my clients gift me.

A bit about myself. Where does photography come into play in my life? Where did I even begin with it? Photography was part of my life since early childhood. My father had a film camera that highly interested me and by the age of 10 my father bought me my first SLR camera. Kids my age were into playing with toys, but I was more interested in a real camera and taking pictures. My first pictures were on black and white film and were developed in a dark room. I started out taking pictures of everything around me. Nature, life events… anywhere I’d go I’d have a camera on hand and my friends were always my photo models. As time would pass friends would ask me to photograph their weddings, with my love for photography, I was excited to be given such opportunities.

I also had different full time and side jobs as a photographer when my wife and I lived in New York City. Within the years, more professional and personal opportunities became available to me opening more interest and experience for me to excel with my love for photography. I came to the point where I really wanted to take my photography more seriously as opposed to a hobby. I began to watch educational videos and read a lot of books on different tips and tricks on lights, and flashes and editing. Everything that would better me as a photographer and help me branch out. This is where I am now.

I now have acquired a professional camera - Nikon D810 - what I consider the best of the best and it has allowed me to excel so much more than I have ever done so before. More opportunities have come for me. From newborn sessions, weddings, engagement sessions, first birthday sessions, maternity photos, family photos, birthdays and variety of events… you name it. I enjoy taking such pictures. Being able to capture a single moment that sometimes you can never relive again. Freezing time in one shot. I love that I am able to do that, and help people treasure such moments and memories. My opinion is that as a photographer, one must always have room for improvement. One must always strive to better and better his work and art. New ideas have to constantly keep coming to life. Even if you become an amazing photographer at the peak of your career, you must always leave room for improvement and try to better yourself. I love my work, and I love what I do. I think that is the key to any career or anything you do in life. Every client I meet is such an honor for me to be able to get a glimpse into their life and be able to capture beautiful moments. So with that being said, I will end this saying that I put a piece of my soul into every photo I take, and I hope that if you do choose me as your photographer - I hope to never disappoint.
</div>
`;
